src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const key = Object.values(robot);
+  const val = Object.keys(robot);
```

Comment:
it's better to call arrays in the prural form 
```suggestion
  const vals = Object.values(robot);
  const keys = Object.keys(robot);
```

Reply:
About plural form agree, but about meaning, in this task, we must put reverse: keys > values, values > keys;